Updated act_command to match new protocol

rschloen opened the pull request:
Names, values, and calculations for act_command fields were updated to
match the new firmware protocol.

Several fields were renamed to avoid confusion. Ex. OP_mode -> command
since OP_mode now refers to different bytes in the command.

A timing block field was removed to match new firmware, and math was changed
since each timing block now consists of 3 bytes.

rschloen committed:
Updated act_command to match new protocol

names, values, and calculations for act_command fields were updated to
match the new firmware protocol.

Several fields were renamed to avoid confusion. Ex. OP_mode -> command
since OP_mode now refers to different bytes in the command.

A timing block field was removed to match new firmware. And math was changed
since each timing block now consists of 3 bytes.

rschloen committed:
Added Set Timing button

Set Timing button sets the cmd_op, which is used in the back end, to
set only the timing blocks without sending actuators.

Made cmd_op writable in the store.

trueb2 commented on src/routes/Manual.svelte:
```diff
@@ -76,27 +76,28 @@ function handleClickRfPower() {
-$: $single_pulse_block = [single_pulse_duration & 0x000000ff,(single_pulse_duration & 0x0000ff00) >> 8,0,0];
+$: $single_pulse_block = [(single_pulse_duration & 0x00000ff0) >> 4,(single_pulse_duration & 0x0000000f) << 4 | (single_pulse_pause & 0x00000f00)>>8,single_pulse_pause & 0x000000ff];
 //set freq to 1kHz (start with 200Hz)
 $: hfOn = hfPeriod * (hfDutyCycle/100);
-$: $hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
+$: $hf_block = [(hfOn & 0x00000ff0) >> 4, (hfOn & 0x0000000f) << 4 | (hfPeriod & 0x00000f00)>>8, hfPeriod & 0x000000ff];
 
 $: lfOn = lfPeriod * (lfDutyCycle/100);
-$: $lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
+$: $lf_block = [(lfOn & 0x00000ff0) >> 4, (lfOn & 0x0000000f) << 4 | (lfPeriod & 0x00000f00)>>8, lfPeriod & 0x000000ff];
```
It would be nice to comment on what the protocol is expecting. What is the intent of this bit math? For example, Grab the top bits of this and use it as the top byte of that.

rschloen committed:
Fixed setting cmd_op

Used temp to reset $cmd_op when setting timing

Added comment to explain timing

rschloen committed:
Added button to turn on/off rf field

onclick sends command with state(on or off).

rschloen committed:
Fixed hexCache and single pulse conditions

Fixed condition on resetting hexCache for interval to include both
single pulse options.

rschloen committed:
Removed unneeded display of variable

rschloen committed:
Added variable for act_cnt8 in store

For future support for larger devices.
